pages/api/submitForm.ts

```diff
@@ -0,0 +1,121 @@
+import { type NextApiRequest, type NextApiResponse } from 'next';
+import { contactSubject } from '../../config';
+import z from 'zod';
+
+// Zod Schema
+interface FormData {
+  Name: string;
+  Subject: string;
+  Email: string;
+  Message: string;
+}
+const formDataSchema = z.strictObject({
+  Name: z.string().max(200),
+  Subject: contactSubject,
+  Email: z.string().max(200),
```

Comment:
if were allowing emails this long, we should just set it to 256 as per https://www.rfc-editor.org/errata/eid1003